highfidelity/hifi
# Add hips pinning / Sit script

This PR adds the ability to pin the hips of the avatar and a new sit script utilizing it.
Add some modifications to teleport.js so avatars can teleport into a chair.
For users in desktop mode, text indicating they can sit on hover when they are within 5m of the chairs and the chair is free.

## Test plan:
- General smoke test of the avatars movements/IK
- Go to hifi://Paris/chair
    - Make sure 2 avatars can't sit in the same chair.
    - If you are virtually and physically sitting and you then stand up in the real world or move too much away from the chair, your avatar should snap out of the chair.
